src/object/srv_obj_migrate.c

```diff
@@ -1065,18 +1067,31 @@ migrate_fetch_update_single(struct migrate_one *mrone, daos_handle_t oh,
 				       mrone->mo_epoch, ORF_EC,
 				       start_shard, 1,
 				       true, false, NULL);
+		los[i].cs_even_dist = 1;
+		los[i].cs_bytes = obj_ec_singv_cell_bytes(
+					mrone->mo_iods[i].iod_size, oca);
+		los[i].cs_nr = obj_ec_tgt_nr(oca);
 	}
 
 	rc = daos_csummer_csum_init_with_packed(&csummer, &csum_iov_fetch);
 	if (rc != 0)
 		D_GOTO(out, rc);
 
-	tmp_csum_iov = csum_iov_fetch;
-	rc = daos_csummer_alloc_iods_csums_with_packed(
-		csummer,
-		&mrone->mo_iods[0],
-		mrone->mo_iod_num,
-		&tmp_csum_iov, &iod_csums);
+	if (!DAOS_OC_IS_EC(oca)) {
```

Comment:
above should be "if (DAOS_OC_IS_EC(oca))"?  (no !)

Reply:
ah, right.